internal/engine/ingester/artifact/artifact.go

```diff
@@ -56,7 +57,7 @@ type verification struct {
 	IsVerified        bool   `json:"is_verified"`
 	Repository        string `json:"repository"`
 	Branch            string `json:"branch"`
-	WorkflowName      string `json:"workflow_name"`
+	SignerIdentity    string `json:"signer_identity"`
```

Comment:
In case someone wonders if it's safe to rename these parameters - yes, they haven't made their way outside staging yet.